cvat/apps/dataset_manager/tests/test_rest_api_formats.py

```diff
@@ -189,12 +291,746 @@ def _generate_url_dump_tasks_annotations(self, task_id):
+                        self.assertEqual(response.status_code, edata['create code'])
+                        data = {
+                            "format": dump_format_name,
+                            "action": "download",
+                        }
+                        response = self._get_request_with_data(url, data, user)
+                        self.assertEqual(response.status_code, edata['code'])
+                        if response.status_code == status.HTTP_200_OK:
+                            content = BytesIO(b"".join(response.streaming_content))
+                            with open(file_zip_name, "wb") as f:
+                                f.write(content.getvalue())
+                        self.assertEqual(osp.exists(file_zip_name), edata['file_exists'])
+
+            # Upload annotations with objects type is shape
+            for upload_format in upload_formats:
+                upload_format_name = upload_format.DISPLAY_NAME
+                if upload_format_name == "CVAT 1.1":
+                    file_zip_name = osp.join(test_dir, f'{test_name}_admin_CVAT for images 1.1.zip')
+                else:
+
+                    file_zip_name = osp.join(test_dir, f'{test_name}_admin_{upload_format_name}.zip')
+                if not upload_format.ENABLED or not osp.exists(file_zip_name):
+                    continue
+                with self.subTest(format=upload_format_name):
+                    if upload_format_name in [
+                        "MOTS PNG 1.0",  # issue #2925 and changed points values
+                    ]:
+                        self.skipTest("Format is fail")
+                    if osp.exists(file_zip_name):
+                        for user, edata in list(expected.items()):
+                            # remove all annotations from task (create new task without annotation)
+                            images = self._generate_task_images(3)
+                            if upload_format_name == "Market-1501 1.0":
+                                task = self._create_task(tasks["market1501"], images)
+                            elif upload_format_name in ["ICDAR Localization 1.0", "ICDAR Recognition 1.0"]:
+                                task = self._create_task(tasks["icdar_localization_and_recognition"], images)
+                            elif upload_format_name == "ICDAR Segmentation 1.0":
+                                task = self._create_task(tasks["icdar_segmentation"], images)
+                            else:
+                                task = self._create_task(tasks["main"], images)
+                            task_id = task["id"]
+                            url = self._generate_url_upload_tasks_annotations(task_id, upload_format_name)
+
+                            with open(file_zip_name, 'rb') as binary_file:
+                                response = self._put_request_with_data(url, {"annotation_file": binary_file}, user)
```

Comment:
@MashaSS , I miss real checks in the test. Basically you create a task, create some annotations, dump the file. The same for upload is done. But the test doesn't check that uploaded and dumped information corresponds to each over.

@yasakova-anastasia , need to design tests which are going to check real data. Not just HTTP codes.